### PR TITLE
improve the constantLR doc

### LINKED ISSUE #120716: `torch.optim.lr_scheduler.ConstantLR`'s description
### 📚 The doc issue

`torch.optim.lr_scheduler.ConstantLR`

> Decays the learning rate of each parameter group by a small constant factor until the number of epoch reaches a pre-defined milestone: total_iters.

Its example:

> ```python
> # Assuming optimizer uses lr = 0.05 for all groups
> # lr = 0.025   if epoch == 0
> # lr = 0.025   if epoch == 1
> # lr = 0.025   if epoch == 2
> # lr = 0.025   if epoch == 3
> # lr = 0.05    if epoch >= 4
> scheduler = ConstantLR(self.opt, factor=0.5, total_iters=4)
> ```

`Decays ... until epoch reaches total_iters` is supposed to decrease LR on the epoch 1, on the epoch 2, ..., on the epoch total_iters, isn't it?

### Suggest a potential alternative/fix

Maybe, `it sets LR to ... until ...`?

cc @svekars @brycebortree

### PR DESCRIPTION
**Pull request by lancerts**
Fixes #120716

### CODE REVIEW

**Comment by lancerts** [comment]
@janeyx99 @albanD can you take a review, thanks

**Review by janeyx99 — APPROVED** [review]
This _is_ more precise.

**Comment by kuraga** [comment]
@lancerts , thanks!
